tests/resume.test.ts

```diff
@@ -82,7 +82,7 @@ describe('upload resume', () => {
       .write();
 
     const resume = FileFactory.pdf(Config.file.MAX_RESUME_FILE_SIZE / 2);
-    const fileLocation = 'fake location';
+    const fileLocation = 's3.amazon.com/upload-resume.pdf';
```

Comment:
nit: instead of defining and using the same `fileLocation` for each route, let's just create a static `FILE_LOCATION` variable at the top of this file set to the url `https://s3.amazonaws.com/upload-resume.pdf` and just use that same one in every test.

Reply:
I think it might be more reasonable to put it within the overall upload describe section since only that section use this specific url.